Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2520 -- Update navigation header to link to subject page

Resolves [EREGCSC-2520](https://jiraent.cms.gov/browse/EREGCSC-2520)

**Description**

Remove Resources and About links from header; add Find by Subject link

**This pull request changes:**

- Updates URLs that are being passes into Vue Single Page Apps from Django templates
   - Removes `about` and `resources` URLs
   - Adds `subjects` URL
- Updates header to only include Statutes and Find by Subjects links
- Updates end to end tests

**Steps to manually verify this change:**

1. Visit [experimental deployment](https://4v3t3q94zf.execute-api.us-east-1.amazonaws.com/dev1184)
2. Notice that the Header only contains two links: 
   - Statutes
   - Find by Subject
1. Visit both of those links from several pages (reader view, Search page, Statutes page, About page, etc)
